CHANGES.md

```diff
@@ -1,5 +1,30 @@
 ## In the next release
```

Comment:
Is this section still necessary?

Reply:
we tend to just leave this at the top so that future changes can go here as they happen. we can remove it and re-add it as needed